contract_discount/__openerp__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 Domatix (<www.domatix.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+{
+    'name': 'Contract Discount',
+    'summary': 'Discounts for contracts and their invoices',
+    'version': '0.1',
+    'author': 'Domatix',
+    'website': 'http://www.domatix.com',
+    'summary': 'Discounts for contracts and their invoices',
+    'depends': ['account_analytic_analysis'],
+    'category': 'Sales Management',
```

Comment:
It,s same category than account_analytic_analysis.

What is the correct category?